### PR TITLE
Fourth milestone completed - Readme & Game Instructions

### PR DESCRIPTION
**Pull request by xtrmdarc**
We kindly request a code review for our fourth milestone, in which we format the readme to include : 

- Instructions on how to play the game
- Instructions on how to run the game
- A live version of the game

Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,71 @@
-# Tic-tac-toe
\ No newline at end of file
+# Tic-Tac-Toe
+
+This a ruby implementation of the game Tic-Tac-Toe. 
+
+# Rules
+
+This is a two-player game. Each player takes turns making placing a token on the board until there's a winner or a draw match.
```
Comment (euqueme) [inline README.md]: 'making placing a token' delete the word 'making' on line #7

File: README.md [hunk 2]
```diff
@@ -1 +1,71 @@
...
+    |4|5|6|
+    |7|8|9|
+
+You can't choose a cell that's not displayed in the figure, that will result in the game displaying a warning and asking you for a proper move.
+
+## Ending the game
+If you want to stop playing, just enter 'n' when asked if you want to keep playing at the end of the match.
+
+# How to run the game
+First of all, you need to have the game files in your computer. You can get them by cloning this repository.
```
Comment (euqueme) [inline README.md]: they can also just download the repo so please explain how

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @xtrmdarc and @ivanid22 :balloon: :lollipop: 

### This project is very professional :star2: you should feel very proud of yourselves, :1st_place_medal:  :1st_place_medal: you actually gave me an idea to make a live demo for my own tic-tac-toe game :smile:  :v: 

### I have a couple of changes for your README file so please address them in the comments below :point_down: 

## [OPTIONAL]

- [x] [OPTIONAL]this wasn't spotted in the previous milestones so I won't request it as mandatory but your game lets you have empty player names, so let's add the validation to correct this

image

### That's it, team! :raised_hands: 

### Please :pray:  do the changes :man_technologist: :man_technologist: and submit back for a code review when you are done :memo: 

Happy coding! :rocket: 

Any question? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

File: README.md (hunk 2 above)
Comment (ivanid22) [reply README.md]: Thanks for your feedback Maria, we just pushed two commits with the fixes you requested

**Review by whiz25 — APPROVED** [review]
## Congz :clap: 
Your README looks professional and it's a :+1:  from me. Merge and proceed.

### Martin Afani

**Pull request merged**